doc/releases/changelog-dev.md

```diff
@@ -280,6 +280,23 @@
           return ([[c, 0.0, 2 * x], [-c, 0.0, 0.0]],)
   ```
 
+* Shots can now be passed as a runtime argument to batch transforms, similarly
```

Comment:
```suggestion
* Shots can now be passed as a runtime argument to transforms that execute circuits in batches, similarly
```

Is this what you mean? Even I don't know what a *batch transform* is, so better make it explicit? :)

Reply:
Oh wait, you mean the actual `batch_transform` function? Maybe use the function name then, so people can look it up?

Reply:
Oh I meant the first one :) Thanks, your suggestion is clearer!